modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition_manager.py

```diff
@@ -49,7 +49,9 @@ def get_objects_from_partitions(cls, partitions):
         list
             The objects wrapped by `partitions`.
         """
-        return DaskWrapper.materialize([partition._data for partition in partitions])
+        return DaskWrapper.materialize(
```

Comment:
Could you add an assertion that `list_of_blocks` has length 1, and also put in the docstring that this function assumes that each partition contains a single block? If someone tries to get objects from partitions that have multiple blocks, we should fail fast instead of just picking up the first block from each partition.

Reply:
The assertion and docstring were added